netlify/functions/webhooks.ts

```diff
@@ -0,0 +1,38 @@
+import { createProbot } from 'probot';
+import { Handler } from '@netlify/functions';
+import type { WebhookEventName } from '@octokit/webhooks-types';
+import app from '../../src/index';
+
+const handler: Handler = async event => {
+  const probot = createProbot();
+  try {
+    await probot.load(app);
+
+    const deliveryHeader = (event.headers['X-GitHub-Delivery'] ||
+      event.headers['x-github-delivery']) as string;
+    const nameHeader = (event.headers['X-GitHub-Event'] ||
+      event.headers['x-github-event']) as WebhookEventName;
+    const sigHeader = (event.headers['X-Hub-Signature-256'] ||
+      event.headers['x-hub-signature-256']) as string;
+
+    await probot.webhooks.verifyAndReceive({
```

Comment:
Haven't quite got this setup yet. It's currently throwing errors:

```
ERROR (event): [@octokit/webhooks] signature does not match event payload and secret
```

I've added the variables to netlify that I know of. But I'll have to dive into the verify function to double check which one is causing the issue.